assets/src/application/ui/display/Tooltip/types.ts

```diff
@@ -0,0 +1,7 @@
+import PopperJs from "popper.js";
```

Comment:
This needs to be in deps

Reply:
chakra depends on this and I was thinking that this is the same

Reply:
btw that `right` placement cause button being overlapped and I've removed that, maybe it's better to let `popper` to choose automatically the best placement:

![image](https://user-images.githubusercontent.com/1477453/85570733-462f0100-b63c-11ea-9034-db5cac8673e6.png)

Reply:
w/o placement:

![image](https://user-images.githubusercontent.com/1477453/85571192-9f973000-b63c-11ea-9f67-f555391957fb.png)